k8s/identity-validation.rego.yaml

```diff
@@ -51,7 +51,7 @@ data:
     pod_name = k8s_claims.pod.name
 
     pod_verified = true {
-        input.service = input.service
+        pod_ns = k8s_claims.namespace
```

Comment:
This line is actually equivalent to the what's currently checked below here:
https://github.com/yahoo/k8s-athenz-identity/blob/7689cbdb69b81f214f65e0162e1d3e5901da7462/k8s/identity-validation.rego.yaml#L59
This will compare the pod namespace is equal to what is in the claim.